docs/VLSI/Tutorial.rst

```diff
@@ -107,6 +105,16 @@ This contains the Hammer configuration for this example project. Example clock c
 
 First, set ``technology.asap7.tarball_dir`` to the absolute path of where the downloaded the ASAP7 PDK tarball lives.
 
+SRAM Compilation
+^^^^^^^^^^^^^^^^
+Hammer can automatically add the collateral for the SRAMs in the ``Sha3RocketConfig``.
+
+.. code-block:: shell
+
+    make srams
+
+This generates a ``generated-src/example.TestHarness.Sha3RocketConfig/sram_generator-output.json`` that will then be consumed by the steps below. Note however, that for the ``Sha3AccelwBB`` that we are pushing the VLSI flow, none of those SRAMs are used.
```

Comment:
We should probably change the example to something that uses SRAMs with an example hand-crafted floorplan.

Reply:
So, Rocket synthesizes to a very large number of SRAMs because the SRAM library is very limited. As a result, the placement constraints would be ridiculously long for a beginner example.

Is there an alternate peripheral to Rocket that has a small memory array?

Reply:
We should make bigger SRAMs, then...

Reply:
Yeah I think the SRAMs should be similar to what you would have in the real-world so somewhere between 2 and 8 SRAMs for a default rocket-chip I$ or D$.